cprover_bindings/src/goto_program/expr.rs

```diff
@@ -1080,6 +1080,14 @@ impl Expr {
     pub fn r_ok(self, e: Expr) -> Expr {
         self.binop(ROk, e)
     }
+
+    // Expressions defined on top of other expressions
+
+    /// `min(self, e)`
+    pub fn min(self, e: Expr) -> Expr {
```

Comment:
If there are side effects, this will evaluate them twice

Reply:
Please take a look at the new changes, which complete the definition for `is_side_effect` and assert that neither of these cause side effects.